### PR TITLE
feat: support named entry in api mode

### PR DESCRIPTION
**Pull request by tmkx**

### CODE REVIEW

**Comment by tmkx** [comment]
Hello, can you review this PR?

File: src/index.ts [hunk 1]
```diff
@@ -336,17 +336,24 @@ const normalizeOptions = async (
...
+      throw new PrettyError(`Cannot find ${input}`)
+    } else {
+      log('CLI', 'info', `Building entry: ${options.entryPoints.join(', ')}`)
+    }
   } else {
-    log('CLI', 'info', `Building entry: ${options.entryPoints.join(', ')}`)
+    if (Object.keys(input).length === 0) {
+      throw new PrettyError(`Cannot find ${JSON.stringify(input)}`)
+    }
+    log('CLI', 'info', `Building entry: ${JSON.stringify(input)}`)
```
Comment (egoist) [inline src/index.ts:356]: you should properly check if the files exist when `input` is an object, now you're just checking if it's empty

**Comment by tmkx** [comment]
https://github.com/egoist/tsup/blob/f7cae2818824838d5ab582bdb50c8b806262ffc0/src/index.ts#L355-L360

if `input` is an array, files' existence will not be checked neither?

**Comment by tmkx** [comment]
> https://github.com/egoist/tsup/blob/f7cae2818824838d5ab582bdb50c8b806262ffc0/src/index.ts#L355-L360
> 
> if `input` is an array, files' existence will not be checked neither?

Sorry, I got it.

**Pull request merged**